include/deal.II/base/tensor.h

```diff
@@ -27,6 +27,14 @@
 #include <ostream>
 #include <vector>
 
+
+#ifdef DEAL_II_WITH_CUDA
+#  include <cuda_runtime_api.h>
+#  define DEAL_II_CUDA_HOST_DEV __host__ __device__
+#else
+#  define DEAL_II_CUDA_HOST_DEV
+#endif
+
```

Comment:
`config.h` is included in this file and that file includes `numbers.h`, so putting the definition at one place should be enough.